File: CustomShare/src/main/java/abhishekti/appyhigh/customshare/models/CustomShareDataText.java
package abhishekti.appyhigh.customshare.models;

/**
 * Created by Abhishek Tiwari on 24-02-2021.
 */
public class CustomShareDataText extends CustomShareData {

    private String text;

    public CustomShareDataText(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }
}

File: CustomShare/src/main/java/abhishekti/appyhigh/customshare/models/CustomShareDataText.kt
package abhishekti.appyhigh.customshare.models

/**
 * Created by Abhishek Tiwari on 02-03-2021.
 */
class CustomShareDataText(text: String) : CustomShareData() {
    private val text: String = text
    fun getText(): String {
        return text
    }

}
